tests/conftest.py

```diff
@@ -92,3 +92,25 @@ async def startup_event():
     app.include_router(fastapi_mock)
 
     return app
+
+@pytest.fixture
+async def fastapi_app_with_allowing_subdomain():
```

Comment:
You actually don't need this new fixture, please see the comment on the file `test_fastapi.py`